lfs.h

```diff
@@ -76,6 +76,7 @@ enum lfs_error {
     LFS_ERR_EXIST       = -17,  // Entry already exists
     LFS_ERR_NOTDIR      = -20,  // Entry is not a dir
     LFS_ERR_ISDIR       = -21,  // Entry is a dir
+    LFS_ERR_NOSYS       = -38,  // Function not implemened
```

Comment:
There is actually a specific error code for read-only filesystems. This would be a good time to use it:

``` c
LFS_ERR_ROFS        = -30,  // Read-only file system
```